### PR TITLE
DCS-840 fixing token refresh

### PR DESCRIPTION
**Pull request by andrewrlee**
Axios upgrade meant that url was no longer recognised which was leading to a POST request to the wrong url leading to a 401

### CODE REVIEW

File: backend/api/oauthApi.js [hunk 1]
```diff
@@ -26,8 +23,7 @@ const oauthApiFactory = (client, { clientId, clientSecret, url }) => {
   const userDetails = (context, username) => get(context, `/api/user/${username}`)
 
   const oauthAxios = axios.create({
-    baseURL: url,
-    url: 'oauth/token',
+    baseURL: `${url}oauth/token`,
```
Comment (andrewrlee) [inline backend/api/oauthApi.js:26]: Tidied up a bit to help a future move to typescript and the ditching of axios for superagent

File: backend/api/systemOauthClient.js [hunk 2]
```diff
@@ -1,34 +0,0 @@
-import Client from './oauthEnabledClient'
```
Comment (andrewrlee) [inline backend/api/systemOauthClient.js:1]: This wasn't being used

File: backend/api/oauthApi.js (hunk 1 above)
Comment (petergphillips) [inline backend/api/oauthApi.js:26]: bit naff that the URL ends in a slash

**Review by petergphillips — APPROVED** [review]
LGTM

File: backend/api/oauthApi.js (hunk 1 above)
Comment (andrewrlee) [reply backend/api/oauthApi.js:26]: yep, I did have a look at fixing it but they all do! 
This was a copy of PSH that we trimmed down.

**Pull request merged**